compiler/cx-zed-extension/docs/language_spec.md

```
@@ -0,0 +1,263 @@
+imported information is implicitly privately declared, so no information imported will be available to any module that imports the current one, as one might expect with #include and a header file.
+
+### 6.2. Visibility Control
+
+The visibility of declarations within a module is controlled by `public:` and `private:` headers.
+
+*   `public:`: All declarations following this header will be visible to other modules that `import` this one.
+*   `private:`: All declarations following this header will be local to the current module and cannot be accessed from outside.
+
+These headers can be used multiple times within a file to toggle the visibility of different sections of code. If no visibility header is specified, declarations are `private` by default.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)